### PR TITLE
Fixed JavaScript message formatting

### PR DESCRIPTION
**Pull request by yogeshsuhagiya**
## This PR is a:

- [ ] New topic
- [x] Content update
- [x] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

Corrected HTML special character issue in Javascript message string.

## Additional information

List all affected URLs 

- https://devdocs.magento.com/guides/v2.3/coding-standards/code-standard-javascript.html

screenshot

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/graphql/develop/resolvers.md [hunk 1]
```diff
@@ -194,7 +194,7 @@ mutation {
 
 A sample response:
 
-```text
+```json
```
Comment (osrecio) [inline guides/v2.3/graphql/develop/resolvers.md]: I think should be as text. @keharper right?

Comment (keharper) [reply guides/v2.3/graphql/develop/resolvers.md]: GraphQL requests should be `text`. The responses should be `json`.  Originally, we had both as `json`, then overcorrected by setting both to `text`. This is the correct way for now.

**Comment by keharper** [comment]
Thanks @yogeshsuhagiya

**Pull request merged**